lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -169,6 +169,10 @@ module InternalConstants
           'Google::Apis::LoggingV2::LogEntryOperation'
         ]
       }
+
+      # The name of the WriteLogEntriesPartialErrors field in the error details.
+      PARTIAL_ERROR_KEY = 'type.googleapis.com/' \
```

Comment:
All of the `*_KEY` constants represent fields in the log record. This is a field in the response, so maybe `PARTIAL_ERROR_FIELD`?

Reply:
Changed to `PARTIAL_ERROR_FIELD`.